### ISSUE TITLE
MSID: 90347 Version: 1 DOI: 2023.07.25.550518

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1708]
"msas": "Neuroscience"
"msid": "90347"
"preprintDoi": "10.1101/2023.07.25.550518"
"Reviewed Preprint posted": "2023-11-17"
"Sent for peer review": "2023-07-19"
"Posted to bioRxiv": "2023-07-28" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2023.07.25.550518v1/")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=90347)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/90347/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.07.25.550518)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/90347/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/90347)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/90347
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/90347
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/90347/v1/90347-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/90347)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by KJT08** [comment]
Hi @fred-atherden the docmap isn't available yet.

**Comment by fred-atherden** [comment]
Thanks @KJT08. It's likely because there's a missing match between the preprint and submission in datahub. I've updated the spreadhseet that manually makes this match, and would expect the docMap to update in the next 24 hours .

**Comment by fred-atherden** [comment]
@hazalCiplak - I've added the link between this MS and the preprint in the [relevant spreadsheet](https://docs.google.com/spreadsheets/d/15QcK8w-ssB7109RQEDtFpJPZ0J5HTGxoHa_2TtpMBbg/edit#gid=1336081641) but the docMap for this one still isn't available. Do you know why that is?

**Comment by HazalCiplak** [comment]
Hi Fred,
Thank you for letting me know and I appreciate you adding the manuscript into the document. 
Typically, it should have been updated already, but the issue arises from some of the previously added data. The problem occurs when the data pipeline is running. The data type of the 'manuscript_id' column should be numeric, and one of the records in the screenshot below is causing problems. I have removed the incorrect characters, so it should run smoothly now.
<img width="659" alt="Screenshot 2023-11-07 at 09 36 41" src="https://github.com/elifesciences/enhanced-preprints-import/assets/6259479/cc615816-8c85-4c28-a33f-bf7030c20e76">

**Comment by fred-atherden** [comment]
Ah I see - thanks for letting me know

**Comment by KJT08** [comment]
Hi @fred-atherden the docmap is there now so I've moved this on.

**Comment by JGilbert-eLife** [comment]
Do we want to fix this, @fred? It's a bioRxiv tagging issue (follows logically from the PDF).


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/1785305/18512bf4-1ded-4a64-b352-85791337bff7)


elocations missing from the references.

**Comment by fred-atherden** [comment]
Thanks @JGilbert-eLife - Yes we want to fix this

**Comment by JGilbert-eLife** [comment]
Yep! Thanks!
